dubbo-plugin/dubbo-qos/src/main/java/org/apache/dubbo/qos/annotation/QosServer.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.dubbo.qos.annotation;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * Prompt to add dubbo qos dependency. If you know qos dependency is needed or it has been added,
+ * you don't have to write this annotation.
+ */
+@Retention(RetentionPolicy.SOURCE)
+@Target(ElementType.TYPE)
+public @interface QosServer {
+}
```

Comment:
Was it better to use a logger to log out when user config `dubbo.application.qos-xxx` if dubbo-qos denendency is absent?

Reply:
`dubbo.application.qos-enable` has default value `true`. Does it make user think qos should start although there is no any qos configuration(system property, dubbo.properties, xml or springboot autoconfigure) in project?

Reply:
ok.

I think it would be better to log out what current environment have / do not have as a info message. Including `dubbo-qos`, `dubbo-registry-xxx`, `dubbo-rpc-xxx`...